docs/release-notes/5205-custom-searcher.txt

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+** New Features **
+
+-  Custom Searcher enables users to implement their own hyperparameter tuning algorithms.
```

Comment:
suggestion: Custom Searcher provides an API for users to implement their own logic to coordinate across multiple training jobs.  This enables easy implementation of custom meta algorithms like hyperparameter tuning and ensembling methods. 

can we link to documentation in the release note?  if so, let's add a link to the docs/tutorial, e.g. "Please take a look at our tutorial to get started."

Reply:
I applied your change. I also added a reference to the user guide. I am not sure about that last part, there seems to be no precedent.